packages/bits-ui/package.json

```diff
@@ -18,7 +18,8 @@
 	"exports": {
 		".": {
 			"types": "./dist/index.d.ts",
-			"svelte": "./dist/index.js"
+			"svelte": "./dist/index.js",
+			"default": "./dist/index.js"
 		}
 	},
 	"files": [
```

Comment:
decided to exclude the exports map for now since it wasn't working quite right with `/utilities`. this can be something that we can add later in a `minor` though.